lib/bootstrap-local.js

```diff
@@ -82,9 +83,7 @@ require.extensions['.ts'] = function (m, filename) {
     // Send it to node to execute.
     return m._compile(result, filename);
   } catch (err) {
-    console.error('Error while running script "' + filename + '":');
-    console.error(err.stack);
-    throw err;
+    console.log(bold(red(err.message)));
```

Comment:
This file is only used in development scenarios and is intended to throw.

Reply:
Understood.  I will restore.

Reply:
@clydin, I am ready to push up the requested changes. However, all my 'net' modifications are now to @angular-devkit/schematics-cli rather than @angular-devkit/core.  Should I change the title of the PR and the consolidating commit to reflect the location change?

Reply:
@clydin , Specifically, the preferred new title and consolidating commit would be:
```
    feat(@angular-devkit/schematics-cli): clean up command-line error messages
```

Reply:
sure